config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -95,10 +95,11 @@ spec:
       ironicAPI:
         replicas: 1
         containerImage: quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo
-      ironicConductor:
-        replicas: 1
+      ironicConductors:
+      - replicas: 1
         containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
         pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+        storageRequest: 10G
```

Comment:
If this passes CI (it works for me locally) then please apply the changes to the other sample https://github.com/openstack-k8s-operators/openstack-operator/blob/master/config/samples/core_v1beta1_openstackcontrolplane_collapsed_cell.yaml  as well to keep that usable.

Reply:
Ack, done.